src/fa-screenshot.sh

```diff
@@ -190,14 +171,10 @@ printOut "-- Checking image properties"
 printFileAttribs
 
 printOut "-- Resizing screenshot"
-convert=`which convert`
-$convert -geometry 1600x ${absFile} ${absFile}
-
-printOut "-- Compressing png"
-compressUsingTinyPNG
+magick=`which magick`
```

Comment:
If magick is in the path, why bother fully qualifying it?

Reply:
Note that the github action shrink images uses tinypng right now too. https://github.com/FusionAuth/fusionauth-site/blob/main/.github/workflows/shrink-images.yml as does the `shrink-images.sh` script.

Reply:
Yeah, the `which` bit seems redundant.